### DISCUSSION TITLE
How to keep terminal windows when refresh browser?

### DISCUSSION THREAD

**Discussion by jiungbaek** [gitpod-io/openvscode-server#356] [Q&A]
Hi.

When I using terminal and refresh a browser tab, session is still connecting, but terminal windows disapper.

How can I keep terminal windows as before refresh?

Theia can keep terminal window when it refreshed

**Comment by jeanp413** [comment]
You mean that the terminal panel is sometimes hidden after reload but it was visible before you reload? seems like a bug upstream

**Reply by jiungbaek** [reply]
Do you have any plan to fix this issue?

**Reply by jeanp413** [reply]
Could you create an issue upstream https://github.com/microsoft/vscode/issues, I'll investigate when I have some free time but they could fix  it before me :slightly_smiling_face:
